source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,190 @@
+=================
+Change a Document
+=================
+
+.. default-domain:: mongodb
+
+In this page, you can learn how to change documents in a MongoDB collection
+using two distinct operation types: 
+
+- :ref:`Update <update-operation>`
+- :ref:`Replace <replace-operation>`
+
+Update operations specify the field and value to change in one or more
+documents. A replace operation specifies the fields and values to replace
+a single document from your collection. 
+
+In the following examples, a paint store sells five different
+colors of paint. The ``paint_inventory`` collection represents their
+current inventory: 
+
+.. code-block:: json
+
+    { "_id": 1, "color": "red", "qty": 5 }
+    { "_id": 2, "color": "purple", "qty": 8 }
+    { "_id": 3, "color": "yellow", "qty": 0 }
+    { "_id": 4, "color": "green", "qty": 6 }
+    { "_id": 5, "color": "pink", "qty": 0 }
+
+.. _update-operation:
+
+Update
+------
+
+Update operations can change the field and value. They apply changes
+specified in an update document to one or more documents that match your
+query filter. 
+
+The :java-docs:`updateOne() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateOne(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+method changes the first document your query filter matches and the
+:java-docs:`updateMany() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateMany(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+method changes all the documents your query filter matches. 
+
+You can call the ``updateOne()`` and ``updateMany()`` methods on a
+``MongoCollection`` instance as follows: 
+
+.. code-block:: java
+
+    collection.updateOne(query, updateDocument);
+
+    collection.updateMany(query, updateDocument);
+    
+Update Operation Parameters:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- ``query`` specifies the document you want to update in your collection
+- ``updateDocument`` specifies the field and value to change in the matching document or documents. For this example, we use the :doc:`Updates builder </fundamentals/builders/updates>` to create the update document.
```

Comment:
1. To my understanding, whether it's updateOne or updateMany, only one field and value gets updated.  updateOne and updateMany only differ based on the number of documents that get updated.
2. I choose not to mention updateOptions since that's more specific to using upsert (and is mentioned on the upsert page) and it's not being used here